### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by Zamiell** [facebook/docusaurus#4182]
Hello,

When using Docusaurus, I got the following error message:

```
Error: `useThemeContext` is used outside of `Layout` Component. See https://v2.docusaurus.io/docs/theme-classic#usethemecontext.
```

This appears to be coming from the following line of source code:

https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-theme-classic/src/theme/hooks/useThemeContext.ts#L17

This link of `https://v2.docusaurus.io/docs/theme-classic#usethemecontext` is broken, so I have no idea what to do next.

**Comment by vamsi3** [comment]
#4183 I made a PR to fix the broken link.

**Comment by lex111** [comment]
Thank for the report, it has been fixed in #4183.
